docs/concepts/executor/serve.md

```diff
@@ -124,18 +119,9 @@ with Deployment(uses=MyExec, port=12345, replicas=2) as dep:
 ```
 ````
 
-The {class}`~jina.orchestrate.deployments.Deployment` class accepts configuration options similar to 
-{ref}`Executor configuration with Flows <flow-configure-executors>`.
-
-````{admonition} See Also
-:class: seealso
-
-For more details on these arguments and the workings of a Flow, see the {ref}`Flow section <flow-cookbook>`.
-````
-
 (kubernetes-executor)=
 ## Serve via Kubernetes
-You can generate Kubernetes configuration files for your containerized Executor by using the static `Executor.to_kubernetes_yaml()` method. This works like {ref}`deploying a Flow in Kubernetes <kubernetes>`, because your Executor is wrapped automatically in a Flow and uses the very same deployment techniques.
+You can generate Kubernetes configuration files for your containerized Executor by using the static `Executor.to_kubernetes_yaml()` method:
```

Comment:
```suggestion
You can generate Kubernetes configuration files for your containerized Executor by using the static `Deployment.to_kubernetes_yaml()` method:
```

Reply:
Changing all instances. What do I change this to?:

```
{meth}`~jina.serve.executors.BaseExecutor.to_kubernetes_yaml()` 
```

Reply:
{meth}`~jina.Deployment.to_kubernetes_yaml()`